content/en/observability_pipelines/configuration/install_the_worker/_index.md

```diff
@@ -126,6 +126,8 @@ The Observability Pipelines Worker supports all major Kubernetes distributions,
 
 See [Update Existing Pipelines][5] if you want to make changes to your pipeline's configuration.
 
+**Note**: If you enable [disk buffering][6] for destinations, you must enable Kubernetes [persistent volumes][7] in the Observability Pipelines helm chart to handle back pressure when a destination is unavailable or can't keep up with the volume of data that the Worker is sending.
```

Comment:
Sorry, reading this again. It reads like the persistent volumes are responsible for handling/alleviating back pressure, when in reality it is the buffer that does that (the PV is necessary to support the buffer but isn't the mechanism which helps with back pressure) -- does that make sense? I feel like we should move "to handle back pressure when a destination is unavailable or can't keep up with the volume of data that the Worker is sending" to after "disk buffering" maybe? I tried to write a suggestion but it felt ugly 😆 so I leave to the experts.